frontend/views/components/modal/Modal.vue

```diff
@@ -1,6 +1,6 @@
 <template lang='pug'>
   div
-    component(:is='content' ref='content')
+    component(:is='content' ref='content' v-bind='contentProps')
```

Comment:
@taoeffect and @pieer. Here (#560) I initially proposed to pass "specific data" to a component through URL query. But with that approach can limit us if we need to pass not primitive values in the future.
Then, I found another way (the one made here) to pass that data as props (which sounds better to me, because it's more "Vue'ish").

What do you think?

Reply:
That seems reasonable to me, although @pieer would know the most Vue-correct way of doing this

Reply:
Sounds good to me.